torch/optim/lr_scheduler.py

```diff
@@ -74,8 +74,13 @@ def wrapper(*args, **kwargs):
         self._step_count = 0
         self.verbose = verbose
 
+        self.reset_optimizer_lr()
         self.step()
 
+    def reset_optimizer_lr(self):
+        for group in self.optimizer.param_groups:
+            group["lr"] = self.optimizer.defaults["lr"]
```

Comment:
Hm I don't think this works because we need to be able to specify a different lr per param group:
https://github.com/pytorch/pytorch/blob/39470387cf17896cf38f3e04b5b81d4647734275/test/test_optim.py#L1980-L1986